docker/compose/ci/docker-compose.yml

```diff
@@ -21,8 +21,16 @@ services:
   app:
     image: "richie:${CIRCLE_SHA1}${IMAGE_SUFFIX}-dev"
     env_file: env.d/ci
-    volumes:
-      - ./dist:/app/dist
     depends_on:
       - "db"
       - "elasticsearch"
+    tmpfs: /data
+
+  # The "package" service defines volumes that SHOULD pre-exist before mounting
```

Comment:
not clear this explanation :-(
I understand that you are starting the image with another user so that you can generate the dist. 
My first idea would be: do we really need docker-compose for this as it does not require the db... ? Just using docker run would be easier to understand and avoid an akward docker-compose conf?